Pod/VOKNavigationHelper/VOKNavigationHelper.m

```diff
@@ -0,0 +1,122 @@
+//
+//  VOKNavigationHelper.m
+//  VOKUtilities
+//
+//  Created by Brock Boland on 1/24/17.
+//  Copyright (c) 2017 Vokal. All rights reserved.
+//
+
+#import "VOKNavigationHelper.h"
+
+@implementation VOKNavigationHelper
+
++ (void)clearExistingViewsAndAnimateToViewController:(UIViewController *)viewController
+                                            duration:(NSTimeInterval)duration
+                                             options:(UIViewAnimationOptions)options
+                                          completion:(void (^ __nullable)(BOOL finished))completion
+{
+    // Enforce the non-null
+    NSParameterAssert(viewController != nil);
+    if (viewController == nil) {
+        return;
+    }
+    
+    id<UIApplicationDelegate> appDelegate = UIApplication.sharedApplication.delegate;
+    
+    if (!appDelegate.window) {
+        appDelegate.window = [[UIWindow alloc] initWithFrame:UIScreen.mainScreen.bounds];
+        appDelegate.window.screen = UIScreen.mainScreen;
+        [appDelegate.window makeKeyAndVisible];
+    }
+    UIWindow *appWindow = appDelegate.window;
+    
+    UIView *snapshot = [UIScreen.mainScreen snapshotViewAfterScreenUpdates:NO];
+    
+    // Get rid of old VC's and views
+    [appWindow.subviews makeObjectsPerformSelector:@selector(removeFromSuperview)];
+    
+    if (appWindow.rootViewController.presentedViewController) {
+        [appWindow.rootViewController dismissViewControllerAnimated:NO
+                                                         completion:^{
+                                                             appWindow.rootViewController = viewController;
+                                                         }];
```

Comment:
Is it going to be problematic that this is asynchronous and potentially happening in parallel to the animation below?